Ladybird/CMakeLists.txt

```diff
@@ -201,6 +201,7 @@ add_dependencies(ladybird SQLServer WebContent WebDriver WebSocketServer Request
 
 function(create_ladybird_bundle target_name)
     set_target_properties(${target_name} PROPERTIES
+        OUTPUT_NAME "Ladybird"
```

Comment:
Since you changed the Output name of the executable, (which is good), you also need to change the CFBundleExecutable (I think?). Or at least, it would be better consistency to do so.